### ISSUE TITLE
Bro fails due to kafka module

### ISSUE THREAD

**Issue by seven62** [rocknsm/rock#272]
`$ broctl check`

```
logger scripts failed.
error in /usr/share/bro/site/scripts/rock/plugins/kafka.bro, line 25: unknown identifier Kafka::logs_to_send, at or near "Kafka::logs_to_send"
```
`$ yum install bro-plugin-kafka`
```
bro-plugin-kafka-0-1.20170803git0d735b5.el7.centos.x86_64 already installed and latest version
The contents of the plugin script loads a init.bro which is not in the directory?
```

`$ cat /usr/lib64/bro/plugins/APACHE_KAFKA/scripts/Apache/Kafka/__load__.bro`

```
#  Licensed to the Apache Software Foundation (ASF) under one or more
#  contributor license agreements.  See the NOTICE file distributed with
...
@load ./init.bro
```

contents of the directory are:  

`$ ls -l`

```
-rw-r--r--. 1 root root  951 Aug  1 21:51 __load__.bro
-rw-r--r--. 1 root root 1475 Nov  5  2017 logs-to-kafka.bro
```

Not 100% sure what is broken in the chain.

**Comment by dcode** [comment]
This should be fixed with the latest bro-plugin-kafka `1:0.2-2.20180317git1dfc523.el7` and the development branch of `rock-scripts`

**Comment by dcode** [comment]
Re-open if this is still an issue.
